js-pkg/create-y-sweet-app/src/templates/nextjs/.env.default

```diff
@@ -0,0 +1 @@
+CONNECTION_STRING="y-sweet-connection-string-goes-here"
```

Comment:
Noticing that this env var name differs from the one in `.env.local`. Is that an accident?

Reply:
This was meant to be a template for their "production" `.env` file from before I switched to linking to the Vercel/Remix docs. I'll remove it!